release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -427,6 +429,15 @@ Starting in {product-title} {product-version}, Operator Lifecycle Manager (OLM)
 Kubernetes 1.22 introduces link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122[several notable changes] to `v1` of the `CustomResorceDefinition` API.
 ====
 
+[id="ocp-4-9-fb-catalogs"]
+==== File-based catalogs
+
+File-based catalogs are the latest iteration of the catalog format in Operator Lifecycle Manager (OLM). It is a plain text-based (JSON or YAML) and declarative config evolution of the earlier, and now deprecated, xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-sqlite-catalogs-deprecated[SQLite database format], and it is fully backwards compatible. The goal of this format is to enable Operator catalog editing, composability, and extensibility.
+
+For more information about the file-based catalog specification, xref:../operators/understanding/olm-packaging-format.adoc#olm-file-based-catalogs_olm-packaging-format[Operator Framework packaging format].
+
+For instructions about creating file-based catalogs by using the `opm` CLI, see xref:../operators/admin/olm-managing-custom-catalogs.adoc#olm-creating-fb-catalog-image_olm-managing-custom-catalogs[Managing custom catalogs].
```

Comment:
For these links, would it make sense to use the title of the heading they jump to rather than the page title?

Reply:
I've started doing this sometimes when the target section sounded redundant in the sentence, and in case the assembly title gave some context as to what sort of topic they might be getting directed to. But I'll still take another look.